Floofbot/Handlers/CommandHandler.cs

```diff
@@ -63,6 +63,11 @@ private async Task HandleCommandAsync(SocketMessage s)
             {
                 var result = await _commands.ExecuteAsync(context, argPos, _services);
 
+                if (result.Error == CommandError.UnknownCommand && msg.HasMentionPrefix(_client.CurrentUser, ref argPos) && msg.ToString().EndsWith("?")) {
+                    string response = Floofbot.Modules.Fun.QuestionEightBall(msg.ToString());
+                    await context.Channel.SendMessageAsync(response);
```

Comment:
Nice update here.
Sanity check: do we want to return after this, or run the rest of the code in this method?

Reply:
You're absolutely right. Does it sound better to move this `if` block after the `Log.Error` block and `await` there?

Reply:
I moved the EightBall response to after the error logging.